Sources/Core/FeaturePrelude/Loadable.swift

```diff
@@ -171,6 +171,49 @@ extension Loadable {
 		flatMap { .success(transform($0)) }
 	}
 
+	public func filter<Element>(by predicate: (Element) -> Bool) -> Loadable<[Element]> where Value == [Element] {
```

Comment:
or:

```swift
	public func filter<S, E>(by predicate: (E) -> Bool) -> Loadable<S> where Value == S, S: Sequence, E == S.Element {
```

To support any kind of `Sequence` ?

Reply:
Should probably be just:
```
public func filter(by predicate: (Value.Element) -> Bool) -> Loadable<[Value.Element]> where Value : Sequence
```